packages/amplify-category-api/src/provider-utils/awscloudformation/utils/rds-secrets/database-secrets.ts

```diff
@@ -109,7 +109,7 @@ export const testDatabaseConnection = async (config: RDSConnectionSecrets) => {
   try {
     await adapter.initialize();
   } catch(error) {
-    printer.error('Failed to connect to the specified RDS Data Source. Check the connection details in the schema and re-try. Use "amplify api update-secrets" to update the user credentials.');
+    printer.error('Failed to connect to the specified RDS Data Source. Check the connection details in the schema. Use "amplify api update-secrets" to update the user credentials and run "amplify api generate-schema" to import the schema.');
```

Comment:
Phani, correct me if I'm wrong- isn't this run `amplify import api` to import the schema?

Reply:
Yes